src/data/faqData.js

```diff
@@ -0,0 +1,51 @@
+export const faqData = [
+  {
+    question: "What is Netflix?",
+    answer:
+      "Netflix is a streaming service that offers a wide variety of" +
+      "award-winning TV shows, movies, anime, documentaries, and more on thousands of" +
+      " internet-connected devices." +
+      "\n" +
+      "You can watch as much as you want, whenever you want " +
+      "without a single commercial – all for one low monthly price. There's always something" +
+      " new to discover and new TV shows and movies are added every week!",
```

Comment:
why this is not just a line string?

Reply:
It was looking bad so i made it multiple lines